utility/services/crypto_to_fiat.py

```diff
@@ -77,11 +80,17 @@ def get_cogs_amount(crypto, fiat, fiat_rate):
         if rate is None:
             return CRYPTO_FIAT_CONVERSION['CURRENT_AGI_USD_RATE']
         else:
-            return round((1 / rate) * fiat_rate)
+            rate = rate.crypto_rate
```

Comment:
please do post the rates you have derived in ropsten

Reply:
1 AGI = $ 0.38461547, is the new rate. We have only a day's data and based on what we derived.